tests/RobotFramework/tests/cumulocity/telemetry/thin-edge_device_telemetry.robot

```diff
@@ -32,6 +32,13 @@ Thin-edge devices support sending custom events
     Log    ${events}
 
 
+Thin-edge devices support sending large events
+    ${large_text}=    Evaluate    "'$(printf '=%.0s' {1..1000000})'"
+    Execute Command    tedge mqtt pub tedge/events/largeEvent '{ "text": "Large event", "large_text_field": ${large_text} }'
```

Comment:
Use `te/device/main///largeEvent` topic instead of `tedge/events/...`. Already replaced the `tedge/events...` topics in [the main branch](https://github.com/thin-edge/thin-edge.io/blob/main/tests/RobotFramework/tests/cumulocity/telemetry/thin-edge_device_telemetry.robot).

Reply:
I rebased and updated the tests. Though I still left a test to cover the translation from the `tedge/` to `te/` topics for oversized messages to ensure everything is converted correctly.